tests/model_serving/model_runtime/vllm/conftest.py

```
@@ -68,22 +93,40 @@ def vllm_inference_service(
     gpu_count = request.param.get("gpu_count")
     timeout = request.param.get("timeout")
-    identifier = ACCELERATOR_IDENTIFIER.get(accelerator_type, Labels.Nvidia.NVIDIA_COM_GPU)
+    #identifier = ACCELERATOR_IDENTIFIER.get(accelerator_type, Labels.Nvidia.NVIDIA_COM_GPU)
     resources: Any = PREDICT_RESOURCES["resources"]
-    resources["requests"][identifier] = gpu_count
-    resources["limits"][identifier] = gpu_count
+    if gpu_count:
+        identifier = ACCELERATOR_IDENTIFIER.get(
+                accelerator_type, Labels.Nvidia.NVIDIA_COM_GPU
+        )
+    #resources: Any = PREDICT_RESOURCES["resources"]
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)